source/common/http/conn_manager_impl.cc

```diff
@@ -1666,12 +1666,13 @@ void ConnectionManagerImpl::ActiveStream::onResetStream(StreamResetReason reset_
   //       3) The codec RX a reset
   //       4) The overload manager reset the stream
   //       If we need to differentiate we need to do it inside the codec. Can start with this.
-  ENVOY_STREAM_LOG(debug, "stream reset", *this);
+  const absl::string_view encoder_details = response_encoder_->getStream().responseDetails();
```

Comment:
`responseDetails` may return an empty string. I think you should check for that and either put something like "N/A" or skip `response_details` all together.

Reply:
We do similar thing here https://github.com/envoyproxy/envoy/blob/b41adf18d0419d4c34cc153bf53b3d27295068a3/source/common/router/router.cc#L1279 - So if it is empty, it will be logged as empty I guess

Reply:
agree it would be better to log a "-" or "N/A" by `encoder_details.empty() ? absl::string_view{"-"} : encoder_details`